ConsoleApp/School/Student.cs

```diff
@@ -0,0 +1,7 @@
+namespace ConsoleApp.School;
+
+class Student : Person
+{
+        public string gender { get; set;}
+        private int score { get; set; }
```

Comment:
code convention violated here
```suggestion
        private int Score { get; set; }
```